features/schools/secondary_subjects.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Secondary subjects
+  So candidates know what secondary subjects we offer for school experience
+  As a school administrator
+  I want to specify what subjects we offer for school experience
+
+  Background: I have completed the wizard thus far
+    Given I am logged in as a DfE user
```

Comment:
Minor, but I'd be tempted to roll this into a more-descriptive step. `Given I'm up to step X of the wizard` and chain them in the step definition.